Unified Alerting's Rollback Warning should be more accurate

If you migrate to Unified Alerting and then roll it back by changing `GF_UNIFIED_ALERTING_ENABLED = false`, you'll get a panic with the below message:
```
panic: New alert rules created while using unified alerting will be deleted, 
set force_migration=true in your grafana.ini and try again if this is okay.
```

You see this message even if no new alert rules have been created since migrating. This message is in place to warn users that they can lose new data. We don't have a bulletproof way to know whether any new rules have been created since migrating, so we always want the user to set `force_migration`. Change the error message to be more clear.

--- a/pkg/services/sqlstore/migrations/ualert/ualert.go
+++ b/pkg/services/sqlstore/migrations/ualert/ualert.go
@@ -75,7 +75,7 @@ func AddDashAlertMigration(mg *migrator.Migrator) {
 	case !mg.Cfg.UnifiedAlerting.IsEnabled() && migrationRun:
 		// Safeguard to prevent data loss when migrating from UA to LA
 		if !mg.Cfg.ForceMigration {
-			panic("New alert rules created while using unified alerting will be deleted, set force_migration=true in your grafana.ini and try again if this is okay.")
+			panic("Grafana has already been migrated to Unified Alerting.\nAny alert rules created while using Unified Alerting will be deleted by rolling back.\n\nSet force_migration=true in your grafana.ini and restart Grafana to roll back and delete Unified Alerting configuration data.")
 		}
 		// Remove the migration entry that creates unified alerting data. This is so when the feature
 		// flag is enabled in the future the migration "move dashboard alerts to unified alerting" will be run again.

